modules/eco-node-health-check-operator-about.adoc

```diff
@@ -6,11 +6,11 @@
 [id="about-node-health-check-operator_{context}"]
 = About the Node Health Check Operator
 
-The Node Health Check Operator deploys the `NodeHealthCheck` controller to detect the health of a node in the cluster. The `NodeHealthCheck` controller creates the `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health. 
+The Node Health Check Operator deploys the `NodeHealthCheck` controller to detect the health of a node in the cluster. The `NodeHealthCheck` controller creates the `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health.
```

Comment:
Not related to name change, but still: there are operators which deploy more controller(s), but some operators don't. They are already the controller which does the actual work. NHC is one of them. So I think "The Node Health Check Operator detects the health of a node in the cluster." would be better.

Reply:
@prabinovRedhat Since this comment is outside the scope of this PR, kindly file a bug to get this addressed.
cc: @slintes